code/game/machinery/smartglass.dm

```diff
@@ -127,5 +127,11 @@
 		
 		if("toggle_transparency")
```

Comment:
if("toggle_transparency", "toggle", "cycle") would make it clearer that they are all the same thing

Reply:
Wasn't sure if that was possible.